docs/cow-protocol/tutorials/widget/widget.md

```diff
@@ -131,24 +131,28 @@ createCowSwapWidget(
+| `env`                  | `CowSwapWidgetEnv`     | 'prod'                 | The environment of the widget (`local` , `prod` , `dev` , `pr`). See [`COWSWAP_URLS`](https://github.com/cowprotocol/cowswap/blob/develop/libs/widget-lib/src/consts.ts) const value for urls.                                                                                                                              |
+| `tradeAssets`          | `TradeAssets`          | Same as in swap.cow.fi | An object containing information about the selling and buying assets. Example: `{ asset: 'WBTC', amount: 12 }` or `{ asset: '0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48' }`.                                                                                                                                                |
+| `theme`                | `CowSwapTheme`         | 'light'                | The theme of the widget (`'dark'` for dark theme or `'light'` for light theme). It is also possible to set your own colors (`CowSwapWidgetPalette`). See [Custom theme](#custom-theme) section for more details.                                                                                                            |
+| `sounds`               | `CowSwapWidgetSounds`  | ---                    | The famous MOOO! sound can be changed to something else, just provide a link to the media file (mp3). You can also specify a value of null to disable the sound.                                                                                                                                                            |
+| `images`               | `CowSwapWidgetImages`  | ---                    | To make your widget unique, you can replace some of the images. For example, a stub image when there are no limit orders.                                                                                                                                                                                                   |
+| `banners`              | `CowSwapWidgetBanners` | ---                    | With this option you can control the display of some banners in the CoW Swap interface.                                                                                                                                                                                                                                     |
+| `tokenLists`           | `string[]`             | ---                    | CowSwap uses the [token lists standard](https://tokenlists.org), which allows flexible and decentralized management of assets for trading.                                                                                                                                                                                  |
```

Comment:
We could maybe add `See also [customTokens] option`  as they are related

Reply:
Fixed